src/api/app/models/user.rb

```diff
@@ -22,6 +22,9 @@ def groups(user)
 class User < ApplicationRecord
   include CanRenderModel
 
+  # Keep in sync with states defined in db/structure.sql
```

Comment:
How about also making this an [enum](http://edgeapi.rubyonrails.org/classes/ActiveRecord/Enum.html) in the model?

Reply:
@hennevogel Been there, done [that](https://github.com/openSUSE/open-build-service/pull/3211). I am okay to give it another try. But not in this PR, and not right now :sweat:

Reply:
But all the things you do then magically work! ;-)

Reply:
Yes, but there were so many from what I remember. So I would prefer to not block this PR for that.